content/docs/tasks/telemetry/distributed-tracing/jaeger/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Jaeger
+description: How to configure the proxies to send tracing requests to Jaeger.
+weight: 10
+keywords: [telemetry,tracing,jaeger]
+---
+
+## Before you begin
+
+*   Setup Istio by following the instructions in the [Installation guide](/docs/setup/).
+
+    Either use the `istio-demo.yaml` or `istio-demo-auth.yaml` template, which includes tracing support, or
+    use the Helm chart with tracing enabled by setting the `--set tracing.enabled=true` option.
```

Comment:
This paragraph is confusing. Which alternative should the reader use? You do not need to provide all possibilities in a task. Pick the one that works best with your task and ask readers to use that. By the end of the sentence I had no clue what option you wanted me to use.